pom.xml

```diff
@@ -67,6 +67,7 @@
     <thrift.version>0.8.0</thrift.version>
     <hypertable.version>0.9.5.6</hypertable.version>
     <couchbase.version>1.1.8</couchbase.version>
+    <tarantool.version>1.6.1-SNAPSHOT</tarantool.version>
```

Comment:
is there a non-SNAPSHOT version we could use here instead? we're starting the release process on Monday and expect it to take no more than 2 weeks. When we release, all versions will have to be non-SNAPSHOT.